src/index.html

```diff
@@ -7,8 +7,33 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header" id="home">
+      <a href="#home" class="way__home">
+        <img src="images\logo.png" alt="moyo_icon"
+          class="icon"
+        >
+      </a>
+      <nav class="nav">
```

Comment:
Add empty lines between multiline sibling blocks of HTML

Reply:
Done